src/indev/lv_indev_private.h

```diff
@@ -89,7 +89,10 @@ struct _lv_indev_t {
         lv_area_t scroll_area;
         lv_point_t gesture_sum; /*Count the gesture pixels to check LV_INDEV_DEF_GESTURE_LIMIT*/
         int32_t diff;
-
+        /*Short click streaks*/
+        uint8_t short_click_streak : 2;
+        lv_point_t last_short_click_point;
+        uint32_t last_short_click_timestamp;
```

Comment:
Let's keep the bitfields close to each other.
```suggestion

        lv_point_t last_short_click_point;
        uint32_t last_short_click_timestamp;        
        /*Short click streaks*/
        uint8_t short_click_streak : 2;
```

Reply:
With the proposed getter for `short_click_streak` that can be used instead of cycling, this can no longer be a bitfield.